dashboard/test/factories.rb

```diff
@@ -42,7 +42,7 @@
 
   factory :user do
     birthday Date.new(1991, 3, 14)
-    sequence(:email) {|n| "testuser#{n}@example.com.xx"}
+    sequence(:email) {|n| "testuser#{n}_#{SecureRandom.hex}@example.com.xx"}
```

Comment:
I'm not crazy about padding these with a random string to avoid collisions. They could still have collisions (randomly) if run enough times, and it will be difficult to predict or debug.

We might instead want to provide explicit user emails for these tests and overwrite them each time like with the facilitator and organizer.

We could do something like add a [trait](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#traits) to user called `for_pd_ui_test`, which creates users with a common email sequence root that can be bulk deleted with a like clause, or add a transient `email_sequence_root` attribute which, where present, is used in place of `'testuser'` in the existing email sequence. 

What do you think?